src/modules/remote.json

```diff
@@ -0,0 +1,174 @@
+{
+  "openrpc": "1.2.4",
+  "info": {
+    "title": "Remote",
+    "description": "A module for managing Remote keys, paring and programming.",
+    "version": "0.0.0"
+  },
+  "methods": [
+    {
+      "name": "list",
+      "summary": "Provides a list of remote controls paired to the device.",
+      "params": [],
+      "result": {
+        "name": "remoteList",
+        "summary": "Contains a list of remote controls paired to the device.",
+        "schema": {
+          "$ref": "#/components/schemas/RemoteControlState"
+        }
+      },
+      "examples": [
+        {
+          "name": "Remote List Example",
+          "params": [],
+          "result": {
+            "name": "Remote List example",
+            "value": {
+              "list": [
+                {
+                  "model": "PR2",
+                  "batteryPercentage": 70
+                }
+              ]
+            }
+          }
+        }
+      ]
+    },
+    {
+      "name": "startPairing",
+      "summary": "Setup the device to allow Remote control to be paired on the next keypress.",
+      "params": [],
+      "result": {
+        "name": "startPairingResult",
+        "summary": "The state of currently paired remote",
+        "schema": {
+          "const": null
+        }
+      },
+      "examples": [
+        {
+          "name": "Remote Pairing Success Example",
+          "params": [],
+          "result": {
+            "name": "Remote State Example",
+            "value": null
+          }
+        }
+      ]
+    },
+    {
+      "name": "onRemotePairing",
```

Comment:
Suggest changing this to `onPairing`.

Keep in mind it'll have a module namespace: `Remote.onPairing` in the full RPC.

Reply:
Hmm, maybe this should be `onPaired`?

Does this event happen once the remote is successfully paired?

Reply:
Changed it form onRemotePairing to onPairing. 

onPaired seems to represent only a successful pairing use case. Keeping in mind the Remote pairing could fail due to 
1. Hardware errors on remote or Device.
2. Not pressing the keys in remote(specific to RDK)
3. RF Interference for RF4ce remotes.

onPairing being present continuous gives us relatively more scope to predict failure if they happen.